pkgs/applications/blockchains/polkadot/default.nix

```diff
@@ -7,16 +7,16 @@
 }:
 rustPlatform.buildRustPackage rec {
   pname = "polkadot";
-  version = "0.9.9-1";
+  version = "0.9.10";
 
   src = fetchFromGitHub {
     owner = "paritytech";
     repo = "polkadot";
     rev = "v${version}";
-    sha256 = "sha256-EmnrwBMHb9jpEZAG393yyMaFRRQJ6YYDRvsp+ATT7MY=";
+    sha256 = "sha256-1iBA7R63g0UOuCmnFONc9j/7jfcHA54jJ327AL/aPIE=";
```

Comment:
You can drop the `sha256-` prefix here.

Reply:
No, you can't. It will complain because the encoding is wrong.